src/loaders/express.js

```diff
@@ -110,19 +109,28 @@ module.exports = async (app) => {
 
   app.use(checkAuthExpiredMiddleware);
 
-  app.use(OpenApiValidator.middleware({
-    apiSpec: path.join(__dirname, '..', 'specs', 'api.yaml'),
-    validateRequests: true,
-    validateResponses: {
-      onError: (error) => {
-        console.log('Response body fails validation: ', error);
-
-        AWSXRay.getSegment().addMetadata('openApiValidationError', JSON.stringify(error));
-        AWSXRay.getSegment().addAnnotation('openApiValidationFailed', true);
-      },
-    },
-    operationHandlers: path.join(__dirname, '..', 'api'),
-  }));
+  ['v1', 'v2'].forEach((version) => {
+    // api and api.v2 could be renamed to api.v1 api.v2 once we focus on data model,
+    // for now leaving like this so we don't get merge conflicts
+    // with other changes that may come in on "api/"
+    const apiYamlFileName = version === 'v1' ? 'api' : `api.${version}`;
```

Comment:
I think this variable is better called `apiVersion`. it doesn't store the filename. The filename is created on line 120.